.eslintrc.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
   rules: {
     "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+    "prettier/prettier": [
+      "error",
+      {
+        endOfLine: "auto",
+      },
+    ],
```

Comment:
needed to get rid of prettier/eslint errors. eslint doesn't like any chars at the end of a line. 
we as team wanted to have chars like , and ; at the end of the line, so this fix ist needed to stick to the team rules